util/bytebuffer-sc.js

```diff
@@ -114,6 +114,11 @@ ByteBuffer.prototype.writeRrsInt32 = function(value, offset) {
     }
     return size;
 };
+*/
+ByteBuffer.prototype.writeRrsInt32 = () => {
+    process.emitWarning("writeRrsInt32 is depreciated.");
```

Comment:
I believe you mean "deprecated" not "depreciated"

Reply:
Everything i do is wrong lol

Reply:
I will fix these typos in the next pr